### ISSUE TITLE
Feature - allow for host OS reboots

### ISSUE THREAD

**Issue by fowie** [microsoft/Windows-Sandbox#4]
PS Script for installing looks great, but what happens if the hos OS reboots? Does the sandbox start up automatically?  If not, I don't think people want to re-run the full installer.  Maybe have the script create a desktop shortcut on the host OS for re-running the sandbox once installed?

**Comment by sebassem** [comment]
Do you mean when the host OS restarts to install the Sandbox feature ? If that's the question , when the user re-runs the script after the installation of the feature the script will check if the installer is already downloaded and just launch the sandbox .

**Comment by fowie** [comment]
Currently, if I restart my desktop, it won't start up the already-installed sandbox automatically, so I'll have to re-run the install script.  That's fine, but might not be clear to most end users, since the script is named "install"

**Comment by brasmith-ms** [comment]
If you have the script run automatically upon logon the Sandbox it should start upon rebooting. I agree that the naming could indicate that the script is install&run to reduce confusion, but the README does explain what the script is doing to help with that.

**Comment by fowie** [comment]
Makes sense.  But there's no instructions in the readme instructing users to make the script run upon logon, so you might get users that run it once, and don't realize after a reboot that it isn't starting by itself.  Maybe there should just be an optional script to run the installer as a scheduled task?

**Comment by haripulapaka** [comment]
brandon, maybe we can have an autorun entry to start the script if a reboot is required?
